lib/TPP/ConvertTppToXsmm.cpp

```diff
@@ -613,8 +611,36 @@ struct ConvertTppReluOp : public OpRewritePattern<tpp::ReluOp> {
     int64_t ldi = *leadDim;
 
     return lowerUnaryTPPtoXSMM(reluOp, rewriter, outputMemRef.getElementType(),
-                          xsmm::UnaryKind::RELU, xsmm::UnaryFlags::NONE,
-                          {m, n, ldi, ldo});
+                               xsmm::UnaryKind::RELU, xsmm::UnaryFlags::NONE,
+                               {m, n, ldi, ldo});
+  }
+};
+
+struct ConvertTppZeroOp : public OpRewritePattern<tpp::ZeroOp> {
+  using OpRewritePattern<tpp::ZeroOp>::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(tpp::ZeroOp zeroOp,
+                                PatternRewriter &rewriter) const override {
+    Type outputType = zeroOp.getInputs()[0].getType();
+    assert(outputType.isa<MemRefType>() && "expect a memref type");
```

Comment:
I think we may want to have this kind of thing in the interface. So that we can assert:
` assert(zeroOp.hasBufferSemantics())`.